app/locale/ja-JP.ts

```diff
@@ -0,0 +1,104 @@
+export default {
+  add: '追加',
+  addMedia: '動画を追加',
+  admin: '管理者',
+  advanced: '上級設定',
+  allow: '許可する',
+  allowTracking: 'アプリの使用状況を送信する',
+  allowTrackingDesc:
+    'アプリの使われ方を調べるために、匿名の使用情報が開発者に送信されます。',
+  avatar: 'アバター画像',
+  avatarCredit: '<0>選択中のアバター画像提供: </0> <1>{{artist}}</1>',
+  cancel: 'キャンセル',
+  changelog: '更新履歴',
+  chatColor: 'チャットの名前の色',
+  chatPlaceholder: 'メッセージ/URL',
+  chatRevealHint: '<1></1> を押すとチャット画面が開きます',
+  chatNewMessage: '新しいメッセージが送信されました！',
+  chooseUsername: 'ユーザーネームを選んでください',
+  connecting: '接続しています',
+  continue: '続ける',
+  copy: 'コピー',
+  copyLink: 'リンクをコピー',
+  debug: 'デバッグ',
+  developerMode: '開発者モード',
+  developerModeDesc: 'ウェブページの検証をする開発者コンソールを開きます。',
+  directIP: 'IPによるダイレクト接続',
+  disallow: '許可しない',
+  disconnect: '切断する',
+  disconnected: '切断しました',
+  discordPresenceDescription:
+    'Discordのプロフィールに情報を表示し、他のユーザーの参加ができるようにします。',
+  dj: 'DJ',
+  donate: '寄付する',
+  donators: '寄付者',
+  duplicate: '複製する',
+  enableDiscordInvite:
+    'Discordから招待するには、設定から <1>Discord Rich Presence</1> を有効にしてください。',
+  endSessionTitle: '公開セッションを終了しますか？',
+  endSessionDescription: '非公開セッションに切り替えると、すべてのユーザーが切断されます。',
+  enterJoinDest: 'フレンドコード/IPアドレスを入力してください。',
+  extensions: '拡張機能',
+  friendCode: 'フレンドコーど',
+  getStarted: '利用開始する',
+  info: '情報',
+  invite: '招待する',
+  join: '参加する',
+  joinSession: 'セッションに参加する',
+  kick: 'キックする',
+  language: '言語',
+  launchDiscordInvite: '招待を送るにはDiscordを起動して、{{productName}} を再起動してください。',
+  lockQueue: '再生リストをロックする',
+  moveToTop: '一番上に移動する',
+  maxUsers: '最大ユーザー数',
+  next: '次の動画',
+  nextUp: '再生リスト',
+  networkDisconnectHostDisconnect: 'ホスト側が接続を終了しました。',
+  networkDisconnectTimeout: 'タイムアウトしました。',
+  networkDisconnectInvalidClientInfo: '本体情報が無効です。',
+  networkDisconnectVersionMismatch: `本体のバージョンが合いません。`,
+  networkDisconnectFull: 'セッションが満員です。',
+  networkDisconnectKicked: 'セッションからキックされました。',
+  networkTroubleshootingHelp: '<1>トラブルシューティングガイド</1> を確認してください。',
+  noticeAddedMedia: '<Media id="{{mediaId}}">{{mediaTitle}}</Media> を追加しました。',
+  noticeMediaError: '{{url}} をリクエスト中にエラーが発生しました。',
+  noticeNowPlaying:
+    '再生中: <Media id="{{mediaId}}">{{mediaTitle}}</Media> requested by <Username id="{{userId}}">{{username}}</Username>.',
```

Comment:
Is it intended, that "requested by" was left in english?

Reply:
wow, thanks for finding a mistake!
ill fix it
but for now im building Metastream myself to test jp translation

i already found some other mistranslation/typo so ill fix them all at same time